R/utils.R

```diff
@@ -179,3 +179,9 @@ file_size <- function(...) {
 prompt_ask_yes_no <- function(reason) {
   utils::menu(c("no", "yes"), FALSE, title = reason) == 2 # nocov
 }
+
+#' @useDynLib storr
+# Read RDS keys fast
+read_text_file <- function(path) {
+  .Call("read_text_file", PACKAGE = "storr", path, nchar)
```

Comment:
This will become

```r
.Call(Cread_text_file, path, nchar)
```

with the changes further down the PR

Reply:
See 4f06d03782e2d7bbe2ee7f35b97e0fcb893dac86.